src/objects.js

```diff
@@ -1,41 +1,74 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name: name,
+          age: age}
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]){
+    return true;
+  } else {
+    return false;
+  }
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65){
+    return true;
+  } else {
+    return false;
+  }
 };
 
 const getAges = people => {
-  // your code here
+  let myArray = [];
+  for (let i = 0; i < people.length; i += 1){
+  myArray.push(people[i].age);
+  };
+  return myArray;
 };
 
 const findByName = (name, people) => {
-  // your code here
+  for (i = 0; i < people.length; i += 1){
+    if (people[i].name === name){
```

Comment:
Great use of for loops! See if you can refactor using an iterator method from Wednesday